common/tools/esm4mocha.mjs

```diff
@@ -32,7 +33,8 @@ export async function load(url, context, defaultLoad) {
     };
   }
   const { source } = await defaultLoad(url, context, defaultLoad);
-  return { format: context.format, source, shortCircuit: true };
+  const format = context.format ?? "builtin";
```

Comment:
node:fs/promises wasn't importing correctly for me. I was thinking it's fine to default to builtin? Alternatively I can also do something like:

```ts
const format = context.format
if (format === undefined && url.startsWith("node:")) format = "builtin"
```

Or if anyone has a better suggestion, I am not too familiar with loader logic

Reply:
interesting! My changes are barely minimum to get many packages test running. I am sure there are cases that aren't handled properly.  If `buildin` works we will use it!